src/rpc/guards/runGuards.ts

```diff
@@ -0,0 +1,23 @@
+import { AGuard } from "@/rpc/guards/AGuard";
+import Rpc from "@/rpc/Rpc";
+import ATransport from "@/transport/ATransport";
+
+/**
+ * Runs guards sequentially.
```

Comment:
I'm not a  fan of leaving these AI comments in the code
this comment is very "[duh](https://dictionary.cambridge.org/dictionary/english/duh)"